packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1388,17 +1469,24 @@ class Container extends StatelessWidget {
 
 // LAYOUT NODES
 
-/// Uses the block layout algorithm for its children.
+/// A widget that uses the block layout algorithm for its children.
 ///
 /// This widget is rarely used directly. Instead, consider using [Block], which
 /// combines the block layout algorithm with scrolling behavior.
 ///
 /// For details about the block layout algorithm, see [RenderBlockBase].
+///
+/// See also:
+///
+///  * [Block] (which combines block layout with scrolling)
```

Comment:
I know I started this trend, but I think we should move from `* foo (which blah)` to `* foo, which blah` (or `*foo, blah` in some cases). The parentheses don't work in all the cases where I want to use this pattern. I can do that in a later pass though.